third-party/qthread/qthread-src/config/qthread_check_hwloc.m4

```diff
@@ -30,7 +30,7 @@ AC_DEFUN([QTHREAD_CHECK_HWLOC], [
   AC_CHECK_HEADERS([hwloc.h],[],
   				   [qt_allgoodsofar=no])
   AS_IF([test "x$qt_allgoodsofar" = xyes],
-	    [AC_SEARCH_LIBS([${with_hwloc_symbol_prefix}topology_init], [hwloc "hwloc -lnuma"], [],
+	    [AC_SEARCH_LIBS([${with_hwloc_symbol_prefix}topology_init], [hwloc "hwloc -lnuma" "hwloc -lnuma -lpciaccess"], [],
```

Comment:
Hmm, I don't love just adding new tries for each hwloc package we enable. I'd much prefer either passing passing in hwloc link args. If you want to stick with this for now, please open a followup issue.

We also need to note this patch in the qthreads README to trigger rebuilds

Reply:
I would rather make this change in a separate PR to avoid complicating this one more. I've updated the README and created issue https://github.com/Cray/chapel-private/issues/4387 to capture modifying `qthreads` configuration to determine `hwloc` dependencies automatically.